### DISCUSSION TITLE
line connected to switch in diagram

### DISCUSSION THREAD

**Discussion by Tinxymaddur** [arendst/Tasmota#21031] [Q&A]
Line connected to switch means our regular power switch ?
Or a separate power switch

**Comment by sfromis** [comment]
That depends on the context and function of the device.

**Comment by NsinghP** [comment]
> Line connected to switch means our regular power switch ? Or a separate power switch

It will be helpfull if you post a picture of the diagram and Manufacturer/ model number of the device.
